src/symbolic.ml

```diff
@@ -33,7 +55,7 @@ module P = struct
 
   type thread = Thread.t
 
-  module Choice = Symbolic_choice.MT
+  module Choice = Choice_monad
```

Comment:
This can be removed if you rename the functor parameter `Choice_monad` to `Choice`.